templates/shared/search/input.tmpl

```diff
@@ -0,0 +1,4 @@
+<!-- Value - value of the search field (for search results page) -->
+<!-- Disabled (optional) - if search field has to be disabled -->
+<!-- Placeholder (optional) - placeholder text to be used -->
+<input type="search" spellcheck="false" name="q" maxlength="255" placeholder="{{if .Placeholder}}{{.Placeholder}}{{else}}{{ctx.Locale.Tr "search.search"}}…{{end}}"{{if .Value}} value="{{.Value}}"{{end}}{{if .Disabled}} disabled{{else}} autofocus{{end}}>
```

Comment:
Maybe the dots in `{{ctx.Locale.Tr "search.search"}}…` should be also put into the translation string, to make it translatable, just like "repo_kind", etc